mm2src/coins/eth.rs

```diff
@@ -876,8 +876,12 @@ pub struct EthCoinImpl {
     decimals: u8,
     history_sync_state: Mutex<HistorySyncState>,
     required_confirmations: AtomicU64,
-    swap_txfee_policy: Mutex<SwapTxFeePolicy>,
+    #[cfg_attr(any(test, feature = "run-docker-tests"), allow(dead_code))]
+    swap_gas_fee_policy: Mutex<SwapGasFeePolicy>,
     max_eth_tx_type: Option<u64>,
+    gas_price_mult: Option<f64>,
+    gas_fee_base_adjust: Option<Vec<f64>>,
+    gas_fee_priority_adjust: Option<Vec<f64>>,
```

Comment:
nit: I would group these gas fields under single type.

Reply:
I grouped gas multipliers in a struct as they are closely related: https://github.com/KomodoPlatform/komodo-defi-framework/pull/2533/commits/62ae0bd728fa820f72555c34ac89fa97e0b9e864